d2l-icon.html

```diff
@@ -0,0 +1,42 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../d2l-colors/d2l-colors.html">
+<dom-module id="d2l-icon">
+	<template>
+		<style include="d2l-colors">
+			:host {
+				color: var(--d2l-color-tungsten);
+				--iron-icon-height: 18px;
+				--iron-icon-width: 18px;
+			}
+			:host([icon^="d2l-tier2:"]) {
+				--iron-icon-height: 24px;
+				--iron-icon-width: 24px;
+			}
+			:host([icon^="d2l-tier3:"]) {
+				--iron-icon-height: 30px;
+				--iron-icon-width: 30px;
+			}
+			:host-context([dir="rtl"]) > iron-icon {
```

Comment:
I think we just want `:host-context([dir="rtl"])` iron-icon, yes?  In almost all cases the `<iron-icon>` will not be a direct child of the context element with `[dir="rtl"]`.